### ISSUE TITLE
Last change register not being persisted.

### ISSUE THREAD

**Issue by elmart** [neovim/neovim#2772]
This is a weird one.
When I type `'.` to go to the last change position, I get `E20: Mark not set`.

Now, I say it's weird because I get this error on current master (3dd3778f098ab33ced3ab3611ecaa600938cf82b) on a new MacBook Pro running OS X Yosemite 10.3.3, but I don't get it on my old MacBook Pro, at the same exact commit, and running the same version of OSX.

So:
- Can anybody else reproduce this?
- Build process signals no error in both cases, and there's nothing of interest in `~/.nvimlog` either. Any ideas about what else could I check?

**Comment by simonweil** [comment]
On same OSX version,
When opening a file and typing `'.`, I get the same error message.

If I do any change and then type `'.`, there is no error.

Hope this helps.

**Comment by elmart** [comment]
Well, yes, it's the same for me. Sorry if I didn't explain it well.
Problem is that "last change" mark seems not to be correctly persisted.
I mean, I do some change, and `'.` works ok. Also, you can see the `.` register set with `:registers`.
Then, you exit nvim, and open it again.
Mark should be there, and `'.` should work, then. But it doesn't. I get the mentioned error then.

**Comment by elmart** [comment]
Ok, mystery solved.
I don't know how, but my `~/.nviminfo` file ended up being owned by root, instead of my normal user. So, no actual reading/writing of it was being done. What I don't understand is how I only noticed such a thing because of that small detail about `'.`. It should have affected many more things...
In any case, I have it working now, and it was my fault, not nvim's.
Sorry for that. Closing.
